extension/src/popup/views/FullscreenSuccessMessage/index.tsx

```diff
@@ -27,21 +26,17 @@ const AvoidScamsWarningBlock = () => {
 
   return (
     <div className="FullscreenSuccessMessage__infoBlock">
-      <Notification
-        variant="warning"
-        icon={<Icon.Warning />}
-        title={t("Avoid scams and keep your account safe")}
-      >
+      <Notification variant="primary" title={t("Keep your account safe")}>
         <div className="InfoBlock__content">
           <ul className="FullscreenSuccessMessage__infoBlock__list">
             <li>
               {t(
-                "Freighter will never ask for your recovery phrase unless you’re actively importing your account using the browser extension - never on an external website.",
+                "Freighter will never ask for your recovery phrase unless you're actively importing your account using the browser extension - never on an external website",
               )}
             </li>
             <li>
               {t(
-                "Always check the domain of websites you’re using Freighter with",
+                "Always check the domain of websites you're using Freighter with",
```

Comment:
should this remain the smart apostrophe to keep with our styleguide?

Reply:
same with line 34

Reply:
:100: I missed this when I copied the text from Figma. Thanks for catching it!